meta/autoid/autoid.go

```diff
@@ -281,11 +286,13 @@ func (alloc *allocator) rebase4Sequence(tableID, requiredBase int64) error {
 		if alloc.sequence.Increment > 0 {
 			if currentEnd >= requiredBase {
 				// Required base satisfied, we don't need to update KV.
+				alreadySatisfied = true
```

Comment:
When the sequence almost runs out, can users rebase to a smaller value?

Reply:
nop